### DISCUSSION TITLE
Transfering Ownership of Fork results in error

### DISCUSSION THREAD

**Discussion by exoad** [community/community#62587] [Repositories]
### Select Topic Area

Bug

### Body

Clicking:

```
Transfer ownership

Transfer this repository to another user or to an organization where you have the ability to create repositories.
```

action button for a **Fork**ed repository and then selecting the organization/user to transfer under the dropdown:

```
New owner

Select one of my organizations
```

Will result in an `Error!`.

Shouldn't the transferring of a fork for ownership just be grayed out entirely?

**Comment by ManishParashar0** [comment]
It seems like you have encountered a bug or unexpected behavior on GitHub when attempting to transfer ownership of a forked repository. If the option to transfer ownership is available for a forked repository and results in an error when selected, it indicates a potential issue with the platform's user interface or logic.

The transfer ownership feature is generally intended for the original repositories, not forks. Transferring a fork could lead to confusion and potential issues with the fork's relationship to the original repository and the source of contributions.

The correct behavior should either be:

1. **The option to transfer ownership should be grayed out for forked repositories**, making it clear that transferring ownership is not applicable to forks.

2. **Or**, if the option is mistakenly available for forked repositories, selecting it should display an error message or warning, explaining that transferring ownership is not allowed for forks.

To address this issue, you should report it to GitHub's support team or submit it as a bug report on their platform. Provide a detailed description of the steps you took, the expected behavior, and the actual outcome (including any error messages) to help GitHub's developers understand and resolve the problem efficiently.

To report the issue, you can visit the GitHub Support page or check their official documentation on how to report bugs. This way, you contribute to improving the platform's usability and user experience for all users.

**Reply by exoad** [reply]
Alright got it! Thank you

**Comment by exoad** [comment]
Alright, this has been posted to the GitHub support!

*This isn't really a platform breaking bug*
